Environment_Engine/Query/FloorGeometry.cs

```diff
@@ -63,7 +63,14 @@ public static Polyline FloorGeometry(this List<Panel> panelsAsSpace, double tole
 
             if (floorGeometry.Count < 1) return null;
 
-            return floorGeometry[0];
+                Polyline max = floorGeometry[0];
```

Comment:
Use a `LINQ` query rather than this `for` loop. If a `for` loop is necessary can it be properly formatted.

Reply:
fix formatting, need advice on LINQ